main.css

```diff
@@ -0,0 +1,725 @@
+  }
+
+  .logout {
+    margin-right: 30px;
+  }
+
+  .nav-item3 {
+    display: none;
+  }
+
+  .web-nav-link {
+    text-decoration: none;
+    color: #272a31;
+    font-size: 25px;
+  }
+
+  .active {
+    border: 3px solid red;
+    padding: 5px;
+  }
+
+  .web-nav-ul {
+    display: flex;
+    list-style-type: none;
+  }
+
+  .web-nav-list {
+    margin-right: 32px;
+  }
+
+  nav.nav-item1 {
+    width: 20%;
+  }
+
+  .nav-item1 img {
+    width: 20%;
+  }
+
+  .program-web {
+    display: flex;
+  }
+
+  .program-item {
+    display: flex;
+    flex-direction: column;
+    width: auto;
+    margin: 1%;
+  }
+
+  .program-item:hover {
+    border: 1px solid #d3d3d3;
+  }
+
+  #devs {
+    display: grid;
+    grid-template-columns: auto auto;
+    padding-right: 10%;
+  }
+
+  .partners img {
+    width: 20%;
+  }
+
+  .logo hr {
+    margin: 1% 45%;
+    border: 2px solid #ec5242;
+  }
+
+  .cn-logo img {
+    width: 20%;
+    border: 1px solid #272a31;
+  }
+
+  .event {
+    display: flex;
+  }
+
+  .event-text {
+    font-size: 35px;
+    text-align: center;
+  }
+
+  .summit {
+    display: flex;
+    height: 20vh;
+    background-size: cover;
+    margin: 30px;
+    background-repeat: no-repeat;
+  }
+
+  .summit div {
+    font-size: 30px;
+    align-self: center;
+    color: white;
+    padding-top: 0%;
+  }
+
+  .first-span {
+    color: blueviolet;
+  }
+
+  .first-header {
+    font-size: 40px;
+    margin-left: 80px;
+  }
+
+  .second-header {
+    font-size: 40px;
+    margin-left: 80px;
+    text-align: center;
+    padding-bottom: 30px;
+  }
+
+  .third-header {
+    font-size: 40px;
+    margin-left: 80px;
+    color: white;
+  }
+
+  .about-description {
+    display: flex;
+    color: #272a31;
+    margin: 5% 10%;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 30px;
+    line-height: 30px;
+    border: 3px solid #d3d3d3;
+    padding: 2%;
+    text-align: center;
+    background-color: #fff;
+  }
+
+  .about-mail {
+    color: #272a31;
+    width: 80%;
+    font-size: 20px;
+    font-weight: normal;
+    line-height: 20px;
+    display: flex;
+    text-align: center;
+    margin: auto;
+    justify-content: center;
+  }
+
+  .about-venue {
+    padding-left: 10%;
+    display: flex;
+    color: #272a31;
+    width: 80%;
+    font-style: normal;
+    font-size: 20px;
+    line-height: 40px;
+    justify-content: center;
+    font-weight: bold;
+  }
+
+  .logo p {
+    padding: 5%;
+    text-align: center;
+    font-weight: bold;
+    font-size: 18px;
+  }
+
+  .title {
+    width: 93%;
+    font-size: 40px;
+    line-height: 70px;
+    display: flex;
+    padding-left: 10%;
+    font-family: 'cocogoose', sans-serif;
+    text-transform: uppercase;
+    opacity: 0;
+    animation: revealText 0.5s forwards;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    background-image: url("images/bg.png");
+    background-size: cover;
+    background-position: center;
+  }
+}
```

Comment:
- [ ] Kindly your page has a scroll bar, kindly fix it to match the original design. And it comes from your main title.

![image](https://user-images.githubusercontent.com/35535130/164777662-75301400-68e7-45eb-a05b-b6ceebe80fd9.png)

![image](https://user-images.githubusercontent.com/35535130/164777721-fc1e4426-a7b2-46b1-9712-0f4dd3794218.png)